### PR TITLE
feat!: Add Android SDK 13.0 support, remove Android SDK 11.0

### PR DESCRIPTION
**Pull request by jeromelaban**
GitHub Issue (If applicable): closes https://github.com/unoplatform/uno/issues/8207

## PR Type

What kind of change does this PR introduce?

## What is the new behavior?
- Support for Android 13
- Removed support for Android 11
- BREAKING CHANGE: `BaseActivity.PrepareOptionsPanel` is now removed as support has been removed by Android 13.0. Use `PreparePanel` instead.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Comment by jeromelaban** [comment]
This PR needs a new build environment to support Android 13, should be available soon.

**Comment by jeromelaban** [comment]
> I am not very convinced of this change. There are many industrial devices that do not support Android 11, I guess those that suuportnao the Android 13 are even rarer.

There's not much convincing to have, this change only moves to the Android 13 SDK, it does not force apps to run on Android 13 devices. Uno still supports Android 5 as it is.

File: src/Directory.Build.targets [hunk 1]
```diff
@@ -97,21 +97,21 @@
...
   </ItemGroup>
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'net6.0-android'">
 	<PackageReference Update="Xamarin.AndroidX.Legacy.Support.v4" Version="1.0.0.10" />
-	<PackageReference Update="Xamarin.AndroidX.AppCompat" Version="1.3.1.3" />
-	<PackageReference Update="Xamarin.AndroidX.RecyclerView" Version="1.2.1.3" />
-	<PackageReference Update="Xamarin.AndroidX.Lifecycle.LiveData" Version="2.3.1.3" />
-	<PackageReference Update="Xamarin.AndroidX.Fragment" Version="1.3.6.3" />
-	<!-- <PackageReference Include="xamarin.androidx.vectordrawable" Version="1.1.0" excludeAssets="buildTransitive" />-->
+	  <PackageReference Update="Xamarin.AndroidX.AppCompat" Version="1.3.1.3" />
```
Comment (dr1rrb) [inline src/Directory.Build.targets:110]: alignment 😝

File: src/Uno.UWP/Extensions/PermissionsHelper.cs [hunk 2]
```diff
@@ -36,7 +36,9 @@ internal static void Initialize(FuncAsync<string, bool> getter, FuncAsync<string
 		public static bool IsDeclaredInManifest(string permission)
 		{
 			var context = Application.Context;
+#pragma warning disable CS0618 // Type or member is obsolete
```
Comment (dr1rrb) [inline src/Uno.UWP/Extensions/PermissionsHelper.cs:39]: Should we open an issue for all those deprecated members?

**Review by dr1rrb — APPROVED** [review]

File: src/Uno.UWP/Extensions/PermissionsHelper.cs (hunk 2 above)
Comment (jeromelaban) [reply src/Uno.UWP/Extensions/PermissionsHelper.cs:39]: Most of the obsolete members continue to work properly, so we'll eventually change those if the APIs end up being removed (which is unlikely to happen, given android's history).

**Pull request merged**